Source/WebInspectorUI/UserInterface/Views/DOMNodeDetailsSidebarPanel.js

```diff
@@ -546,6 +547,14 @@ WI.DOMNodeDetailsSidebarPanel = class DOMNodeDetailsSidebarPanel extends WI.DOMD
                         checked = WI.UIString("No");
                 }
 
+                var switchState = "";
+                if (accessibilityProperties.switchState !== undefined) {
+                    if (accessibilityProperties.switchState == InspectorBackend.Enum.DOM.AccessibilityPropertiesSwitchState.On)
+                        switchState = WI.UIString("On");
+                    else
+                        switchState = WI.UIString("Off");
+                }
```

Comment:
rather than explicitly checking for `undefined` it'd be much simpler to do this
```js
// COMPATIBILITY (iOS X.0): DOM.AccessibilityProperties.switchState did not exist yet.
if (InspectorBackend.Enum.DOM.AccessibilityPropertiesSwitchState) {
    switch (accessibilityProperties.switchState) {
    case InspectorBackend.Enum.DOM.AccessibilityPropertiesSwitchState.On:
        switchState = WI.UIString("On");
        break;
    case InspectorBackend.Enum.DOM.AccessibilityPropertiesSwitchState.Off:
        switchState = WI.UIString("Off");
        break;
    }
}
```